src/pages/dar_application/DataUseAgreements.js

```diff
@@ -15,7 +15,7 @@ const StepAlertTemplate = (props) => {
 
   return (
     ul({style: ulLinkStyle}, [
-      li({isRendered: props.step1Invalid}, [
+      li({isRendered: props.step1Invalid || props.libraryCardInvalid}, [
```

Comment:
Works properly with missing library card, so this comment is potentially out of scope for this ticket: 

When my profile is incomplete (I removed part of my address), I get the alert on the Step 1 page itself to complete my profile, but I do not get this link at the end of the dar application to fix Step 1. I think `props.step1Invalid` doesn't check for all ways a profile may be incomplete. Not sure on the intended behavior, but I thought I would mention it.

Reply:
I can reproduce this behavior in dev, so I'm going to file a separate bug ticket for: 
> When my profile is incomplete (I removed part of my address), I get the alert on the Step 1 page itself to complete my profile, but I do not get this link at the end of the dar application to fix Step 1. I think props.step1Invalid doesn't check for all ways a profile may be incomplete.

Reply:
See https://broadworkbench.atlassian.net/browse/DUOS-1685